stylus-sdk/src/storage/bytes.rs

```diff
@@ -14,6 +15,7 @@ use core::cell::OnceCell;
 pub struct StorageBytes {
     root: U256,
     base: OnceCell<U256>,
+    __stylus_host: *const dyn crate::host::Host,
```

Comment:
Is this better than storing a Box? I'm afraid we are adding complexity here but the generated code could be the same.